runtime/onert/core/src/exec/Executors.cc

```diff
@@ -41,13 +64,13 @@ const ir::OperandInfo Executors::inputInfo(const ir::IOIndex &index)
     // TODO handle general case
     const auto desc = _model_edges->pkg_inputs[index.value()];
     const auto model_idx = std::get<0>(desc);
-    const auto executor_idx = ir::SubgraphIndex{model_idx.value()};
+    const auto executor_idx = ExecutorIndex{model_idx.value()};
```

Comment:
@hseok-oh Does it assume executor order is same to model order? That is, executor 1 comes from model 1's subgraph 1. executor 2 comes from model 2's subgraph, ... ?